src/components/ActionItem/ActionItem.test.js

```diff
@@ -17,37 +17,37 @@ describe('ActionItem unit test', () => {
   });
 
   it('Should apply given className class', () => {
-    const { getByTestId } = render(ActionItem, { propsData: { ...defaultProps } });
+    const { getByTestId } = render(ActionItem, { props: { ...defaultProps } });
```

Comment:
Isn't `{ props: defaultProps }` the same? (Just asking, I see that you just replaced the name of the object property)

Reply:
I think is because I'm making a copy of `defaultProps`, not just passing it through, or maybe is to clarify how to extent props